recipes/pxblat/meta.yaml

```diff
@@ -1,50 +1,60 @@
-{% set name = "pxblat" %}
-{% set version = "0.2.0" %}
+{% set name = "PxBLAT" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "86a2414757113136c3171d77c58d440a196afb349c118b7a9a9d8bd70cb3ff39" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: https://github.com/cauliyang/pxblat/archive/v{{ version }}.tar.gz 
-  sha256: ac8132e9a33854963f42e027e7f2a5280ac516076b109cd07cacd26745a16c7b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pxblat-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 1
-  skip: True  #[py2k or win]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  number: 0
+  skip: True  # [py < 39]
```

Comment:
I think the preferred way is to remove this skip here and instead have `- python >=3.9` in host and run.

Reply:
Thanks for pointing this out, I've made edits accordingly in my next commit.